docs/workers.md

```diff
@@ -531,6 +531,26 @@ the stream writer for the `presence` stream:
 
     ^/_matrix/client/(api/v1|r0|v3|unstable)/presence/
 
+#### Restrict outbound federation traffic to a specific set of workers
```

Comment:
Should we add some information about this to the worker section of the [config manual](https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html)?

Reply:
It's already cross-linked from `outbound_federation_restricted_to` in the config documentation which is in the "Coordinating workers" section

Reply:
Sounds good to me!